Pointwise.lua

```diff
@@ -25,13 +27,22 @@ local zero = torch.FloatTensor({0});
 
 function Pointwise:updateOutput(input)
    self:createIODescriptors(input)
+   local activDesc = ffi.new('struct cudnnActivationStruct*[1]')
```

Comment:
move this to a function, and cache it in self, i.e.:
self.activDesc = self.activDesc or ffi.new(...)

Then, add an entry to the clearDesc function.